### PR TITLE
update mapping

### PR DESCRIPTION
**Pull request by morkusporkus**

### CODE REVIEW

File: src/Server/Features/OfferMatching/List.cs [hunk 1]
```diff
@@ -64,7 +66,7 @@ public ListMappingProfile()
 
         CreateMap<Player, OfferMatchingListResult.OfferMatchingItem>()
             .ForMember(d => d.Team, mo => mo.MapFrom(s => s.Team.Name))
-            .ForMember(d => d.OfferingTeam, mo => mo.MapFrom(s => s.Team != null ? s.Team.Name : string.Empty))
+            .ForMember(d => d.OfferingTeam, mo => mo.MapFrom(s => s.Bids.FindHighestBid()!.Team.Name != null ? s.Team.Name : string.Empty))
```
[the remark this answers is not in the document]
Comment (morkusporkus) [reply src/Server/Features/OfferMatching/List.cs:69]: There was no test, thats why it didn't fail

[the remark this answers is not in the document]
Comment (Lyddon7522) [reply src/Server/Features/OfferMatching/List.cs:69]: Best way to pass a test, is to not have a test 🤣

[the remark this answers is not in the document]
Comment (benjaminsampica) [reply src/Server/Features/OfferMatching/List.cs]: there's a test its just bad :( https://github.com/benjaminsampica/DynamoLeagueBlazor/blob/d0ed6b8c64df6625786201713fb5ac395bb52872/src/Tests/Features/OfferMatching/ListTests.cs#L35

[the remark this answers is not in the document]
Comment (morkusporkus) [reply src/Server/Features/OfferMatching/List.cs]: updated

**Review by benjaminsampica — APPROVED** [review]

**Pull request merged**